### ISSUE TITLE
viewUsers.php should be in admin rights array, or else cannot create new user

### ISSUE THREAD

**Issue by tonythomas01** [Igalia/phpreport#268]
This happened after we flushed out contents from $admin array in https://github.com/Igalia/phpreport/commit/7cc144c47c6160e054a2c8195c2b491bca9db06f

**Comment by tonythomas01** [comment]
Fixed in 363946c115b93438824618083bb149add1f6a7b3

```
commit aa964cb698d42999ddf0b245f7b90be3d364b1a7
Merge: 7cc144c 363946c
Author: Tony Thomas <01tonythomas@gmail.com>
Date:   Sat Jul 23 11:24:04 2016 +0530

    Merge pull request #269 from tonythomas01/fixAdminRights
    
    [#268] Re-add viewUsers.php to admin array, allow managing users

commit 363946c115b93438824618083bb149add1f6a7b3
Author: Tony Thomas <01tonythomas@gmail.com>
Date:   Sat Jul 23 11:15:30 2016 +0530

    [#268] Re-add viewUsers.php to admin array, allow managing users`
```

**Comment by jaragunde** [comment]
Added an extra commit to fix the `IsAdmin` calls. That's why I couldn't find that usage of `isAdmin` when I was grepping the code.
```
commit ca478f199f6998c18f9bfc6368120a1fe7080e30
Author: Jacobo Aragunde Pérez <jaragunde@igalia.com>
Date:   Sat Jul 23 10:54:08 2016 +0200

    [#268] Fix IsAdmin calls instead of isAdmin, sort permissions.php.
```
